filament/backend/src/vulkan/VulkanDisposer.cpp

```diff
@@ -17,10 +17,15 @@
 #include "vulkan/VulkanDisposer.h"
 
 #include <utils/debug.h>
+#include <utils/Log.h>
 
 namespace filament {
 namespace backend {
 
+// Always wait at least 3 frames after a DriverAPI-level resource has been destroyed for safe
+// destruction, due to potential usage by outstanding command buffers and triple buffering.
+static constexpr uint32_t FRAMES_BEFORE_EVICTION = 3;
```

Comment:
Do we need to do something special now in `VulkanDriver::terminate` (like call `mDisposer.gc()` 3 times) to ensure that all resources are actually destroyed?

Reply:
Good question: terminate calls `mDisposer.reset()` which has been updated so that it now ensures that all resources are actually destroyed.